providers/google.go

```diff
@@ -60,6 +59,9 @@ type GoogleOp struct {
 	Scopes                    []string
 	RedirectURIs              []string
 	GQSign                    bool
+	OpenBrowser               bool
```

Comment:
why is OpenBrowser exposed but httpClient, not exposed. They should probably both be not exposed or is there some value is setting them directly?

Reply:
Good point. Yeah I should be consistent here and expose it. Not sure why I didn't do that.

Reply:
Fixed here: 58f9751